### PR TITLE
build: Harden flaky Aeron tests in CI

### PR DESCRIPTION
**Pull request by patriknw**
* increase /dev/shm and use that (by default)
* use default term buffer size
* increase cpu requests, shouldn't matter but corresponds to what we want to use, 2 pods per node

This looks very promising. I have tried in a gke cluster. Verified with `df -h`. It was 64 MB and now 1G.

No more "Scheduled sending of heartbeat was delayed".

This wasn't possible when we tried last time https://github.com/akka/akka/issues/30601

### CODE REVIEW

File: .github/workflows/multi-node.yml [hunk 1]
```diff
@@ -147,7 +144,8 @@ jobs:
           gcloud components install gke-gcloud-auth-plugin
           gcloud config set compute/region us-central1
           gcloud config set compute/zone us-central1-c
-          ./kubernetes/create-cluster-gke.sh "akka-artery-aeron-cluster-${GITHUB_RUN_ID}"
+          gcloud container clusters get-credentials akka-artery-aeron-cluster-test --zone us-central1-c --project akka-team
+          # ./kubernetes/create-cluster-gke.sh "akka-artery-aeron-cluster-test"
```
Comment (octonato) [inline .github/workflows/multi-node.yml:148]: is this intentional? Not calling the script to create the cluster?

File: .github/workflows/nightly-builds.yml [hunk 2]
```diff
@@ -215,8 +215,6 @@ jobs:
           -Dakka.test.tags.exclude=gh-exclude,gh-exclude-aeron,timing \
           -Dakka.test.multi-in-test=false \
           -Dakka.cluster.assert=on \
-          -Daeron.dir=/opt/volumes/media-driver \
-          -Daeron.term.buffer.length=33554432 \
           clean ${{ matrix.command }}
```
[the remark this answers is not in the document]
Comment (patriknw) [reply .github/workflows/nightly-builds.yml:218]: Plenty of space, no problem.
```
Filesystem      Size  Used Avail Use% Mounted on
/dev/root        84G   62G   22G  74% /
tmpfs           7.9G  172K  7.9G   1% /dev/shm
tmpfs           3.2G  1.1M  3.2G   1% /run
tmpfs           5.0M     0  5.0M   0% /run/lock
/dev/sdb15      105M  6.1M   99M   6% /boot/efi
/dev/sda1        63G  4.1G   56G   7% /mnt
tmpfs           1.6G   12K  1.6G   1% /run/user/1001
```

**Review by pvlugter — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
There was an error: "insuffiient usable storage for new log of ". I have increased it. I don't know if it accumulates when running all tests? It's supposed to delete the files on shutdown.

**Comment by patriknw** [comment]
I separated the aeron run in separate workflow. I hope that shows up so I can trigger a manual run if I merge this?

**Pull request merged**